stdlib/contextvars.pyi

```diff
@@ -35,6 +35,10 @@ def copy_context() -> Context: ...
 # a different value.
 class Context(Mapping[ContextVar[Any], Any]):
     def __init__(self) -> None: ...
+    @overload
+    def get(self, __key: ContextVar[Any]) -> Optional[Any]: ...
+    @overload
+    def get(self, __key: ContextVar[Any], __default: Optional[Any]) -> Optional[Any]: ...
```

Comment:
I don't think a `None` check should be required if you provide a default:

```suggestion
    def get(self, __key: ContextVar[Any], __default: Optional[Any]) -> Any: ...
```

Reply:
...although there's no difference between `context.get(key)` and `context.get(key, None)`, so maybe add `_default: None = ...` to the first overload?